core/README.md

```diff
@@ -71,16 +95,27 @@ supported by this endpoint, e.g., `text/html`.
 | ------------- | ----------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
 | `service-doc` | `/api.html` | OAFeat   | A human-consumable service description. The path for this endpoint is only recommended to be `/api.html`, but may be another path. |
 
-Additionally, `child` relations may exist to individual catalogs and collections.
+Additionally, `child` relations may exist to child Catalogs and Collections and `item` relations to Items. These
+relations form a directed graph that enables traversal from a root catalog or collection to items.
+
+| **rel** | **href** | **From**  | **Description**                        |
+| ------- | -------- | --------- | -------------------------------------- |
+| `child` | various  | STAC Core | The child STAC Catalogs & Collections. |
+| `item`  | various  | STAC Core | The child STAC Items.                  |
+
+While it is valid to have `item` links from the landing page, most STAC API implementations 
+serve large numbers of features, so they will typically use several layers of intermediate `child` links before
+getting to Item objects.  These relations form a directed graph
+of Catalogs and Collections, where interior nodes contain `child` relations, and the penultimate nodes will be
+Catalogs with `item` relations to individual Items as leaf nodes. 
 
-| **rel** | **href** | **From**  | **Description**                                                                                          |
-| ------- | -------- | --------- | -------------------------------------------------------------------------------------------------------- |
-| `child` | various  | STAC Core | The child STAC Catalogs & Collections. Provides curated paths to get to STAC Collection and Item objects |
+`child` link relations may point to another STAC Catalog that *also* acts as a STAC API root, which can support
+search over only a sub-catalog. This is useful for very large or federated catalogs that cannot support searching
+over the entire catalog, but can support searching over individual sub-catalogs within it.
 
-It is also valid to have `item` links from the landing page, but most STAC API services are used to 
-serve up a large number of features, so they typically
-use several layers of intermediate `child` links before getting to Item objects.  Note that the `items` (plural)
-link will be used by APIs implementing STAC API - Features to link from a Collection to the items in that collection.
+Note that there is a different link relation `items` (plural)
+used by APIs conforming to the `STAC API - Features` class that links from a Collection to the items in
```

Comment:
```suggestion
used by APIs conforming to the [`STAC API - Features`](../../ogcapi-features/README.md) class that links from a Collection to the items in
```

(not sure if this will render right, so may need tweaks, but just think it'd be nice to provide a link for people)